packages/shell-api/src/cursor.spec.ts

```diff
@@ -793,6 +793,29 @@ describe('Cursor', function () {
       });
     });
 
+    describe('#toJSON', function () {
+      let spCursor: StubbedInstance<ServiceProviderCursor>;
+      let shellApiCursor: Cursor;
+
+      beforeEach(function () {
+        spCursor = stubInterface<ServiceProviderCursor>();
+        shellApiCursor = new Cursor(mongo, spCursor);
+      });
+
+      it('throws with the .toArray() suggestion', function () {
+        try {
+          shellApiCursor.toJSON();
```

Comment:
Should this use `JSON.stringify()`?

Reply:
Isn't it just the direct calling of the method in the unit test, the one that will be called using JSON.stringify? I have also e2e tests with JSON.stringify().

Reply:
Well, it's generally preferable to have tests that verify externally visible behavior when there's no significant difference in complexity, right?